components/host-dashboard/AddFundsModal.js

```diff
@@ -324,6 +326,7 @@ const AddFundsModal = ({ host, collective, ...props }) => {
       <CollectiveModalHeader collective={collective} onClick={handleClose} />
       <Formik
         initialValues={getInitialValues({ hostFeePercent: defaultHostFeePercent, account: collective })}
+        enableReinitialize={true}
```

Comment:
I am not quite understanding why we need the `enableReinitialize` here? 

To my understanding this will re-initialize the form once the user adds the funds preventing the next step (adding platform tip) from completing since the re-initialization will invalidate the form (the required fields; `amount`, `description` will be empty). Also we don't allow the user to add platform tips if there's a host fee; but this will make the `hostFee=0` and thus will always show the platform tip section in the second step. 🤔

Reply:
@SudharakaP The default `hostFeePercent` might change after the second fetch gets completed, `enableReinitialize` provided an easy way to update the modal in this situation. Is there an easy way to update the step change so that we could keep relying on it?

Reply:
Proposed a fix at, https://github.com/opencollective/opencollective-frontend/pull/7898